packages/suite-desktop/src-electron/libs/menu.ts

```diff
@@ -69,6 +73,11 @@ if (!isDev) {
     mainMenuTemplate[2].submenu.splice(2, 1);
 }
 
+if (isDev) {
```

Comment:
Why to remove it for dev mode? Perhaps you wanted the opposite condition?

Reply:
~~During development, if you restart app, you get electron window without renderer~~

Okay, not true anymore. It was probably fixed by a4b1798ab2491da828f3debb92afe3708304ed8b

Then, I would leave it in both development and production

Reply:
b1aacb4700a36ea410483bf65dbc079dcc88adff